Eggstravaganza/Assets/PlayerInputActions.cs

```diff
@@ -0,0 +1,1179 @@
+//------------------------------------------------------------------------------
```

Comment:
would it be okay if we change the autogeneration location to the scripts folder?

<img width="190" alt="input-actions" src="https://github.com/camillewalters/Eggstravaganza/assets/97626507/759c0586-9ddb-424a-a58c-0cdfe8c89215">